jaxrs-spec/src/main/asciidoc/chapters/resources/_resource_field.adoc

```diff
@@ -59,8 +59,8 @@ to limitations of the built-in `valueOf` method that is part of all Java
 enumerations, a `fromString` method is often defined by the enum
 writers. Consequently, the `fromString` method is preferred when
 available.].
-5.  `List<T>`, `Set<T>`, or `SortedSet<T>`, where `T` satisfies
-1,3 or 4 above.
+5.  `List<T>`, `Set<T>`, `SortedSet<T>` or an array (i.e. `T[]`), where
```

Comment:
Minor comment: I would drop the "i.e." part, and say as in the Javadocs "an array T[]" explicitly.

Reply:
Sounds good.  Fixed.